tests/sca_image/examples/.github/workflows/vulnerable_container.yaml

```diff
@@ -0,0 +1,29 @@
+on: pull_request
```

Comment:
what is this file? when do you use it?

Reply:
It's an example of a github actions file which is referencing an image (that we want to scan)

Reply:
got it